src/scripts/main.js

```diff
@@ -1 +1,51 @@
 'use strict';
+
+const clicks = ['click', 'contextmenu'];
+
+const firstPromise = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error(`First promise was rejected`)), 3000);
+
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+});
+
+const secondPromise = new Promise((resolve) => {
+  clicks.forEach((e) => {
+    document.addEventListener(e, () => {
+      resolve('Second promise was resolved');
+    });
+  });
```

Comment:
The `secondPromise` is set to resolve on any event in the `clicks` array. Ensure it resolves only on left or right clicks as specified in the task description.